includes/REST_API/Embed_Controller.php

```diff
@@ -168,15 +168,39 @@ public function get_proxy_item( $request ) {
 	/**
 	 * Retrieves the story metadata for a given URL on the current site.
 	 *
-	 * @SuppressWarnings(PHPMD.NPathComplexity)
```

Comment:
🎉

Reply:
The complexity is now just in the new method but psst :-)